thunder/executors/torchex.py

```diff
@@ -1133,6 +1134,15 @@ def _tril_transform(a: TensorLike, /, diagonal: int = 0, *, fill_value: None | N
     return tril(a, diagonal)
 
 
+# NOTE PyTorch's triu like tril does not have a fill_value parameter
+def _triu_checker(a: TensorLike, /, diagonal: int = 0, *, fill_value: None | Number = None) -> bool:
+    return fill_value is None
+
+
+def _triu_transform(a: TensorLike, /, diagonal: int = 0, *, fill_value: None | Number = None) -> TensorLike:
```

Comment:
Great!  I think the last thing is that the implementation needs to be registered with this checker and transform like line 1150.

Reply:
Thanks, I should have caught that